### ISSUE TITLE
cannot install uber-juno on Win10 x64 since atom release 1.40

### ISSUE THREAD

**Issue by rryi** [atom/atom#19789]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

Looks similar to #19709. However that issue does not give any details, I will try to give some details.

### Steps to Reproduce

1. 
download atom-x64-windows.zip of release 1.39.1,
unpack to C:\RRtool
delete C:\RRtool\Atom
rename "C:\RRtool\Atom x64" to c:\RRtool\Atom
delete %USERPROFILE%\.atom 
start atom by calling "C:\RRtool\Atom\atom.exe"
Welcome guide -> install a package -> open Installer
search "juno"
install "uber-juno"

2. 
download atom-x64-windows.zip of release 1.40
unpack to C:\RRtool
delete C:\RRtool\Atom
rename "C:\RRtool\Atom x64" to c:\RRtool\Atom
delete %USERPROFILE%\.atom 
start atom by calling "C:\RRtool\Atom\atom.exe"
Welcome guide -> install a package -> open Installer
search "juno"
install "uber-juno"

**Expected behavior:**

uber-juno is sucessfully installed in step 1 as well as in step 2

**Actual behavior:**

step 1, performed just today, ends with a successful installation, as expected.

This happened also in earlier releases. I tried install with the last release before 1.40 only to exclude a regression in uber-juno.

In Step 2, installation displays this error:

```
Juno: Error installing package julia-client

Please check your internet connection, or report this to
    http://discuss.junolab.org
and we'll try to help.
    –––
Error: Installing “julia-client@0.9.4” failed.
```

To get more info, I tried to install julia-client directly. It fails, as expected, and shows the following details. 

```
> @pfitzseb/node-pty-prebuilt@0.8.1-prebuilt3 install R:\Temp\apm-install-dir-119714-10908-9jgein.rbpwc\node_modules\julia-client\node_modules\@pfitzseb\node-pty-prebuilt
> prebuild-install || node scripts/install.js


R:\Temp\apm-install-dir-119714-10908-9jgein.rbpwc\node_modules\julia-client\node_modules\@pfitzseb\node-pty-prebuilt>if not defined npm_config_node_gyp (node "C:\RRtool\Atom\resources\app\apm\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\RRtool\Atom\resources\app\apm\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild ) 

prebuild-install WARN install No prebuilt binaries found (target=3.1.10 runtime=node arch=x64 libc= platform=win32)
gyp ERR! configure error 
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (C:\RRtool\Atom\resources\app\apm\node_modules\npm\node_modules\node-gyp\lib\configure.js:492:19)
gyp ERR! stack     at PythonFinder.<anonymous> (C:\RRtool\Atom\resources\app\apm\node_modules\npm\node_modules\node-gyp\lib\configure.js:517:16)
gyp ERR! stack     at C:\RRtool\Atom\resources\app\apm\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:182:21)
gyp ERR! System Windows_NT 10.0.18362
gyp ERR! command "C:\\RRtool\\Atom\\resources\\app\\apm\\bin\\node.exe" "C:\\RRtool\\Atom\\resources\\app\\apm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd R:\Temp\apm-install-dir-119714-10908-9jgein.rbpwc\node_modules\julia-client\node_modules\@pfitzseb\node-pty-prebuilt
gyp ERR! node -v v10.2.1
gyp ERR! node-gyp -v v3.7.0
gyp ERR! not ok 
npm WARN enoent ENOENT: no such file or directory, open 'R:\Temp\apm-install-dir-119714-10908-9jgein.rbpwc\package.json'
npm WARN apm-install-dir-119714-10908-9jgein.rbpwc No description
npm WARN apm-install-dir-119714-10908-9jgein.rbpwc No repository field.
npm WARN apm-install-dir-119714-10908-9jgein.rbpwc No README data
npm WARN apm-install-dir-119714-10908-9jgein.rbpwc No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @pfitzseb/node-pty-prebuilt@0.8.1-prebuilt3 install: `prebuild-install || node scripts/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @pfitzseb/node-pty-prebuilt@0.8.1-prebuilt3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\RR\.atom\.apm\_logs\2019-08-14T18_39_33_262Z-debug.log
```

**Reproduces how often:**
100% 
(tried 3 times)

### Versions

atom 1.40 
uber-juno v0.2.0, 
Win10 x64, system build 18372.267 (2019 H1)

### Additional Information

I reported the error also as an issue to the juno people. See

https://github.com/JunoLab/uber-juno/issues/73

I cannot decide if the error is to be fixed in atom, or uber-juno (or both).elease.

**Comment by rsese** [comment]
Thanks for the report!  I think this might be related to https://github.com/atom/atom/issues/19780 (e.g. see https://github.com/atom/atom/issues/19780#issuecomment-521539431 and  https://github.com/atom/atom/issues/19780#issuecomment-521775324 and https://github.com/JunoLab/atom-julia-client/issues/590).

But thanks for opening an issue on JunoLab/uber-juno, it's always helpful for the package maintainers to take a look and if possible, share more details on what the specific issue is with Atom (vs. an issue with the package itself).

Let's keep this open for now but you can subscribe to https://github.com/atom/atom/issues/19780 if you'd like and we can see if https://github.com/atom/atom/issues/19780 is resolved, if that helps with this issue as well.

**Comment by rsese** [comment]
@rryi: I believe this is fixed now as of Atom 1.40.1 (see https://github.com/JunoLab/atom-julia-client/issues/590 and https://github.com/atom/atom/issues/19780#issuecomment-523062952) but please let me know if you see otherwise.
